plugins/@grouparoo/klaviyo/src/lib/export/exportRecord.ts

```diff
@@ -0,0 +1,159 @@
+import { getKnownFieldMap } from "./destinationMappingOptions";
+import { Errors, ExportRecordPluginMethod } from "@grouparoo/core";
+import { connect } from "../connect";
+
+async function handleGroupsChanges(client, email, oldGroups, newGroups) {
+  const listMapByName = await client.getListMapByName();
+
+  // Unsubscribe from old groups
+  for (const groupName of oldGroups) {
```

Comment:
This isn't quite right. see how there is some "set" work done in other plugins
`oldGroups` is what it was before (so it could be ['a', 'b'])
`newGroups` is what it is now (so it could be ['b','c'])
The user is still in `b` so we don't want to remove them.

Reply:
Ahaaa, good catch.
Gonna handle it.